### DISCUSSION TITLE
'use client' not working

### DISCUSSION THREAD

**Discussion by parletre-in-the-usa** [vercel/next.js#55091] [Help] [answered]
### Summary

hi, i have a button with an event handler that does a console log ('ok'). however, when i click the button, nothing is logged, neither server side nor client side. i added another console log ('rendering') before the event handler definition, and this one does log but only on the server side. i have 'use client' at the top of the file. shouldn't 'rendering' be logged client side? and why is the event handler on the button not called on click?

### Additional information

```js
'use client'

import React from 'react'

const Home = () => {
  console.log('rendering')
  function handleClick() {console.log('ok')}
  
  return (
    <div>
      <button onClick={handleClick}>button</button>
    </div>
    
  )  
}
export default Home
```
```


### Example

_No response_

**Comment by icyJoseph** [comment, marked as answer]
What Node.js version do you have? We noticed that some Node versions caused this error.  Please try with Node 18.17.1 https://nodejs.org/en

**Reply by parletre-in-the-usa** [reply]
ah! did an update, all is working now. thank you!
